history-service/init.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE submissions (
+	submission_id SERIAL PRIMARY KEY,
+	question_id INT NOT NULL,
+	runtime INT NOT NULL,
+	submission_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	username TEXT,
```

Comment:
I think there should be two usernames per submission, since there's always two users collaborating on any questions. And we'd need to save fewer entries.